### PR TITLE
Add french translate and adjust formSpec size for label

### PR DESCRIPTION
**Pull request by d-stephane**

### CODE REVIEW

File: init.lua [hunk 1]
```diff
@@ -5,6 +5,8 @@ if minetest.get_translator then S = minetest.get_translator(MOD_NAME) end
 
 treasure_chest = {}
 
+-- local S = minetest.get_translator("treasure_chest")
+
```
Comment (ZenonSeth) [inline init.lua]: This should really be removed, not left in, since the translator `S` function is defined above.

**Review by ZenonSeth — CHANGES_REQUESTED** [review]
One minor change, otherwise LGTM!

File: init.lua (hunk 1 above)
Comment (d-stephane) [reply init.lua]: ahhh a forgotten comment....

**Review by ZenonSeth — APPROVED** [review]
Cool, thanks for your contribution - I will make a new release later today and publish it on Content DB

**Pull request merged**